building/target_platforms/target_platforms_updates.adoc

```diff
@@ -95,26 +98,41 @@ If your requirement is not available in an update site (p2 repository), eg. only
 
 Non-p2 artifacts will be added to https://github.com/jbosstools/jbosstools-locus[JBoss Tools Locus] as p2 artifacts, so they can be included in the target platform.
 
-== Create a build.xml to mirror 3rd-party repository
 
-First, clone the http://github.com/jbosstools/jbosstools-download.jboss.org[jbosstools-download.jboss.org] Git repository. Then browse to the `jbosstools/updates/requirements` folder. That's where you'll create your `build.xml`.
+== Determine if dependency is mirrored
+
+The next step to requesting a new bundle be added to the target platform is to determine whether the bundle is already mirrored. JBoss Tools mirrors our requirements at http://download.jboss.org/jbosstools/updates/requirements/
```

Comment:
this is not something that should be required by anyone opening up a req to the TP.

it would be nice if they did, but for sure nothing we want to add in as a must-do for those who do not have easy access.

IMO not something that should be in this document.